Test/ModelTest/UserTest.php

```diff
@@ -14,61 +14,45 @@ class UserTest extends TestCase
      * @var User[]
      */
     private $users;
-    /**
-     * @var User
-     */
-    private $user1;
-    /**
-     * @var User
-     */
-    private $user2;
 
     public function setUp()
     {
         $this->users = [];
```

Comment:
remove this property and setup all necessary objects inside the test, you can use `dataProvider` if tests are repetitive with different data